examples/with-typescript/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "scripts": {
+    "dev": "$(npm bin)/concurrently \"tsc --watch\" \"next \"",
```

Comment:
shouldn't need to add `$(npm bin)/` in package.json scripts--NPM configures `PATH` automatically for you when running npm scripts.

should be safe to just say `"concurrently 'tsc --watch' 'next'"`